src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -538,7 +537,7 @@ private void EnsurePrecedingWhitespaceRemoved(XText whitespace, XNode node, Conv
         {
             if (!String.IsNullOrEmpty(whitespace.Value) && whitespace.NodeType != XmlNodeType.CDATA)
             {
-                var message = testType == ConverterTestType.WhitespacePrecedingEndElementWrong ? "The whitespace preceding this end element is incorrect." : "The whitespace preceding this node is incorrect.";
+                var message = testType == ConverterTestType.WhitespacePrecedingEndElementWrong ? "The whitespace preceding this end element is incorrect." : "The whitespace preceding this comment is incorrect.";
```

Comment:
Again, is the whitespace always before a comment?

Reply:
See above.